sqs_interface_test.go

```diff
@@ -5,11 +5,9 @@ import (
 
 	"github.com/Wattpad/sqsconsumer"
 	"github.com/aws/aws-sdk-go/service/sqs"
-	. "github.com/smartystreets/goconvey/convey"
+	"github.com/stretchr/testify/assert"
 )
 
-func TestSQSInterfaceMatches(t *testing.T) {
-	Convey("sqs.New() should implement SQSAPI", t, func() {
-		So(sqs.New(nil), ShouldImplement, (*sqsconsumer.SQSAPI)(nil))
-	})
+func TestSQSInterfaceImplementsSQSAPI(t *testing.T) {
+	assert.Implements(t, (*sqsconsumer.SQSAPI)(nil), sqs.New(nil))
 }
```

Comment:
How so? It exists simply to ensure that if the AWS library changes the SQS API then the local interface will not match and the test will start failing.